server/src/client_api.rs

```diff
@@ -642,7 +641,10 @@ mod tests {
             .send()
             .await;
         assert!(status.is_ok());
-        assert_eq!(status.unwrap().status(), StatusCode::FORBIDDEN);
+        assert_eq!(
+            status.unwrap().status().as_u16(),
+            StatusCode::FORBIDDEN.as_u16()
```

Comment:
I see that in line 580 StatusCode is prefixed with actix_http but not on this line, is that intended? 
Is it because this StatusCode comes from `reqwest::StatusCode`?

Reply:
Yeah, bumping to actix_http 4.8 seems to make those two separate, I've tried to stick to reqwest status codes for the client and actix for the server (since actix wants to return its own status codes, not reqwests). They should be interchangeable, the http status code spec is complete.